genprotocol/sserver-avdl/sserver.avdl

```diff
@@ -0,0 +1,7 @@
+@namespace("sserver")
```

Comment:
Also, maybe "searchsrv.1" would be more descriptive (otherwise, it's not clear what the first "s" is).